coalib/core/Core.py

```diff
@@ -62,184 +62,6 @@ def group(iterable, key=lambda x: x):
     return zip(keys, elements)
 
 
-def cleanup_bear(bear,
```

Comment:
also it's not official API yet, so until the official release we can move stuff around like wanted.